binding.gyp

```diff
@@ -22,7 +22,6 @@
       '<@(tensorflow_include_dir)/tensorflow/c/c_api.h',
       '<@(tensorflow_include_dir)/tensorflow/c/eager/c_api.h',
     ],
-    'tensorflow-library-action': 'symlink'
```

Comment:
I think we should keep this here - it makes a single replace easier if we continue to pass a command-line option into the staging/moving script. See below.

Reply:
Done.